.changeset/contextPath.md

```diff
@@ -0,0 +1,9 @@
+---
+"@sap-ux/fe-fpm-writer": patch
+"@sap-ux/types": patch
+---
+
+feat: contextPath for new pages
```

Comment:
I'd remove line six, the long text in line 8 is good by itself for the changelog.

Reply:
done